src/helpers/messages.js

```diff
@@ -75,17 +75,11 @@ function getActionMessage(scope, submitButtonLabel = __('OK')) {
         case 'section':
         case 'testSection':
         case 'part':
-            msg = __('Click "%s" to continue', submitButtonLabel).trim();
-            if (msg) {
-                msg += '.';
-            }
-            return msg;
+            return __('Click "%s" to continue.', submitButtonLabel).trim();
```

Comment:
**nitpick:** This will be considered as a new translation token, and the previous one will be removed. It will introduce a regression. I'd advise fixing the related translations, by adding the missing dot to their tokens. Otherwise, they will be lost.

<img width="508" alt="image" src="https://user-images.githubusercontent.com/1500098/161973012-b6deeccc-8be0-4d0c-a81a-8600bfd70c20.png">

<img width="514" alt="image" src="https://user-images.githubusercontent.com/1500098/161973591-aaa2a8c1-32f5-405b-ba61-3b7323860fc0.png">